script.program.fritzact/resources/lib/tools.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+import xbmc
+import xbmcgui
+import xbmcaddon
+import os
+import random
+
+__addon__ = xbmcaddon.Addon()
+__addonID__ = __addon__.getAddonInfo('id')
+__path__ = __addon__.getAddonInfo('path')
```

Comment:
Same thing here: you could use `xbmc.translatePath(xbmcaddon.Addon().getAddonInfo('path')).decode('utf-8')` and then only use it with `os.path.join` for the other paths.